src/components/Dailog.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+  <div class="main_container">
+    <div class="container" v-for="issue in issues" :key="issue.id">
```

Comment:
Please name this issues -> issuedBooks , issue -> issuedBooks and make the changes accordingly